### PR TITLE
[Config] Update test suites for CephAdm Anisble wrapper module

### PR DESCRIPTION
**Pull request by vamahaja**
Fix consists of - 
- Merge CephAdm ansible wrapper bootstrap test suites.
- Fix Polarion ID
- Add test suites in Pipeline metadata for 5.1, 6.0 & 6.1

### CODE REVIEW

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by pranavprakash20 — DISMISSED** [review]
LGTM

File: pipeline/metadata/5.3.yaml [hunk 1]
```diff
@@ -68,6 +68,26 @@ pipelines:
...
+          suite: "suites/pacific/cephadm/tier-1-cephadm-ansible-wrapper-modules.yaml"
+          global-conf: "conf/pacific/cephadm/tier-1_5node_cephadm_bootstrap.yaml"
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - dmfg
+        - name: "Basic operation test suite for Cephadm Ansible Bootstrap Module"
+          execution_time: ""
```
Comment (manasagowri) [inline pipeline/metadata/5.3.yaml:82]: Please add time taken for suite execution.

File: pipeline/metadata/6.0.yaml [hunk 2]
```diff
@@ -61,6 +61,26 @@ pipelines:
...
+          suite: "suites/quincy/cephadm/tier-1-cephadm-ansible-wrapper-modules.yaml"
+          global-conf: "conf/quincy/cephadm/tier-1_5node_cephadm_bootstrap.yaml"
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - dmfg
+        - name: "Basic operation test suite for Cephadm Ansible Bootstrap Module"
+          execution_time: ""
```
Comment (manasagowri) [inline pipeline/metadata/6.0.yaml:75]: Please add time taken for suite execution.

File: pipeline/metadata/6.1.yaml [hunk 3]
```diff
@@ -57,6 +57,26 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - cephfs
+        - name: "Basic operation test suite for Cephadm Ansible Modules"
+          execution_time: ""
```
Comment (manasagowri) [inline pipeline/metadata/6.1.yaml:61]: Please add time taken for suite execution.

**Review by manasagowri — COMMENTED** [review]
Just a few minor changes

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by tintumathew10 — APPROVED** [review]
LGTM

**Pull request merged**